### DISCUSSION TITLE
How about adding package-update-indicator?

### DISCUSSION THREAD

**Discussion by WeirdGnome** [SpiralLinux/SpiralLinux-project#275] [General]
Hi!
New user here. linux user since 2003. 
I just love Love Spiral Linux and the lxqt version just flies on my thinkpad x200.
I have just a small sugestion. How about adding package-update-indicator to desktops like lxqt?
(btw. it needs the gnome-package-updater package installed too)
Its small applet to aply updates and it's available in the debian repos.
Just my thought.
Cheers!

**Comment by geckolinux** [comment]
Hi there, thanks a lot for the suggestion. However I don't think that would be ideal. The Gnome package updater framework is not always reliable, and plus I don't like automatic update nags. This is the recommended GUI update method for SpiralLinux:
https://github.com/SpiralLinux/SpiralLinux-project/wiki#updates-within-a-debian-stable-release-lifecycle
